theme/Style/Style.qml

```diff
@@ -5,12 +5,16 @@ import QtQuick 2.5
 
 import com.nextcloud.desktopclient 1.0
 
-QtObject {
+Item {
+    SystemPalette { id: activePalette; colorGroup: SystemPalette.Active }
     // Colors
+    property color window: activePalette.window

```

Comment:
@FlexW I am no sure the naming is not too concise. I'd say, we need a postfix for each of those. Like, "windowBackgroundColor", "windowsTextColor", "buttonColor"? Or similar. In this specific file, it looks obvious what **button** means, because we can see it is a field within the **activePalette** property, but, from the outside, this may not be always obvious. When we say **Style.button** - what that style is? Is it just a color, or, is it a more complex style that includes multiple fields? Another way would be to use fully-qualified names like **Style.activePalette.button**, **Style.activePalette.windowText**, **Style.activePalette.window**.

Reply:
Fully agree with this. It's probably best to do it via a subobject to get some proper namespacing, so I'd lean toward the second option.